build.rs

```diff
@@ -16,33 +16,70 @@ lazy_static! {
     };
 }
 
-fn main() {
+enum SrcMethod {
+    Cmake(PathBuf),
+    RawStatic(PathBuf),
+    PkgConfig,
+}
+
+fn get_source_method() -> SrcMethod {
+    let src: Option<_> = std::env::var_os("APRILTAG_SRC");
+    let src: Option<String> = src.map(|s| s.into_string().unwrap());
```

Comment:
It always panics if if cannot convert to String. Consider `std::env::var_os("APRILTAG_SRC").unwrap()`.